iceoryx_utils/test/moduletests/test_relative_pointer.cpp

```diff
@@ -13,13 +13,13 @@
 // limitations under the License.
 
 #include "iceoryx_utils/internal/relocatable_pointer/relative_ptr.hpp"
+#include "iceoryx_utils/platform/mman.hpp"
+#include "iceoryx_utils/platform/stat.hpp" /* For mode constants */
 
 #include "test.hpp"
 
 #include <cstring>
 #include <fcntl.h> /* For O_* constants */
-#include <sys/mman.h>
-#include <sys/stat.h> /* For mode constants */
 using namespace ::testing;
```

Comment:
A more serious issue is that the tests are not robust when the OS does not respect the shared memory mapping hint. This seems not to be fixed here but an internal fix is in progress which will later be merged to OSS. (There are still problems with QNX in this fix, so I cannot provide a fix right now...)